client/components/recycleAndReuseComponents/Steps/StepThree/Geolocation/InfoBox.jsx

```diff
@@ -27,7 +28,18 @@ export const InfoBox = ({ content, handleCloseInfoBox }) => (
 				>
 					X
 				</Button>
+				{displayDirection ?
+					<Button
+						mt={1}
+						size="xs">
+							<Link href={`https://www.google.com/maps/dir/?api=1&origin=${homeMarker.current.lat}%2C${homeMarker.current.lng}&destination=${content.props.marker.postal}`} target="_blank">
```

Comment:
for cleanliness:

- pull this link out into a const
- have consts for origin and destination to pass in to the link
- remember to update the access for homeMarker since you would be passing in the value instead of the whole ref (see other comment)